### PR TITLE
fix: android onboarding button positioning

### PR DESCRIPTION
**Pull request by AmarTrebinjac**
## Changes

Button was below the fold due to Android's handling of the address bar. [See example](https://dailydotdev.slack.com/archives/C01L0QXQJNB/p1734270081654579)

After doing some digging around and trying a bunch of things, in the end the solution was to make the position of the wrapping parent fixed + setting max/min dvh, as found on the [chrome developer blog](https://developer.chrome.com/blog/url-bar-resizing)

Working example from my S20 Ultra:
image

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

MI-701 #done

### Preview domain
https://fix-android.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/pages/onboarding.tsx [hunk 1]
```diff
@@ -358,7 +358,12 @@ export function OnboardPage(): ReactElement {
   }
 
   return (
-    <div className="z-3 flex h-full max-h-screen min-h-screen w-full flex-1 flex-col items-center overflow-x-hidden">
+    <div
+      className={classNames(
+        'z-3 flex h-full max-h-screen min-h-screen w-full flex-1 flex-col items-center overflow-x-hidden',
```
Comment (rebelchris) [inline packages/webapp/pages/onboarding.tsx:363]: Technically you want to swap the max-h no?

Something like:
```
isCTA ? 'max-h-dvh' : 'max-h-screen'
```

Actually think maybe you can just use max-h-dvh everywhere no?

**Review by rebelchris — CHANGES_REQUESTED** [review]
Just cleanup query?

File: packages/webapp/pages/onboarding.tsx (hunk 1 above)
Comment (AmarTrebinjac) [reply packages/webapp/pages/onboarding.tsx:363]: > Actually think maybe you can just use max-h-dvh everywhere no?
@rebelchris You mean just replace the max-h-screen / min-h-screen with dvh equivalents entirely?

Comment (rebelchris) [reply packages/webapp/pages/onboarding.tsx:363]: Yeah would that work?
Otherwise above ternary maybe neater?

Comment (AmarTrebinjac) [reply packages/webapp/pages/onboarding.tsx]: @rebelchris That did seem to work, indeed. 

I did the onboarding flow a couple of times to check if all is in order. It seemed to look as it should to me.

**Review by rebelchris — APPROVED** [review]
Looks good, maybe good to validate on prod the onboarding flow again (just safe keeping)

**Review by sshanzel — APPROVED** [review]

**Comment by AmarTrebinjac** [comment]
> Looks good, maybe good to validate on prod the onboarding flow again (just safe keeping)

Tested on my S20 Ultra on prod and everything seemed as it should :)

**Pull request merged**